Block/Info/AbstractInfo.php

```diff
@@ -11,29 +11,32 @@
 
 namespace Adyen\Payment\Block\Info;
 
+use Adyen\Payment\Helper\Config;
+use Adyen\Payment\Model\ResourceModel\Order\Payment\CollectionFactory;
 use Magento\Framework\View\Element\Template;
+use Magento\Framework\View\Element\Template\Context;
 
 class AbstractInfo extends \Magento\Payment\Block\Info
 {
+
     /**
-     * @var \Adyen\Payment\Helper\Data
+     * @var \Adyen\Payment\Helper\Config
      */
-    protected $_adyenHelper;
+    protected $_configHelper;
```

Comment:
Could you change this to `$configHelper`? Also in the rest of the PR

Reply:
Part of this refactoring was done in #2105. Skipping this one to prevent confliction.